iroha/src/block.rs

```diff
@@ -81,6 +88,8 @@ pub struct BlockHeader {
     pub previous_block_hash: Hash,
     /// Hash of merkle tree root of the tree of transactions hashes.
     pub merkle_root_hash: Hash,
+    /// Number of view changes after the previous block was committed and before this block was committed.
+    pub number_of_view_changes: u32,
```

Comment:
This should be extracted from `block` module.

Reply:
What do you think is an appropriate place for this field? Blocks serve right now as common synchronization points - the network topology is basically defined by the hash of the previous block and the number of shifts. 

For the synchronization to work the new peer needs this information to validate signatures. And so the information should be stored. The only thing that is stored right now are blocks. Maybe we should have a call with @takemiyamakoto to discuss this in more detail?

Reply:
Agree - it's a tricky question how to work with it here.